test/spirv-version-controls-negative-2.spt

```diff
@@ -0,0 +1,34 @@
+119734787 1024 393230 12 0·
```

Comment:
Again, there is unusual punctuation at the ends of the lines in this test, which would be good to remove.

Reply:
Fixed, thanks!